### ISSUE TITLE
Cannot get program to function properly/monitors won't align for mouse crossing; also several small UI issues

### ISSUE THREAD

**Issue by just4747** [mgth/LittleBigMouse#288]
Thank you for updating, but having some issues. I can no longer get it to make my two different resolution monitors line up like before (as far as mouse crossing goes).

I tried changing size and position of monitors to try to get them to align and no luck. I don't remember having to manually set these things in the older version either but was always able to get it work easily.

Even after I adjust size and position to seemingly line up the top and bottom of the monitor bezel edges, and Stop and Start the app, the mouse pointer still will not cross monitors in a straight line at all except for at the very top. Doesn't make sense per the config. Both are set to 340mm top to bottom and aligned in the app but the mouse does not act as if they are.

Also the Stop and Start buttons in the UI rarely seem to update appropriately or work. Using the right click on the task bar/quick access icon versions of Stop and Start also seems to not work most of the time. Sometimes unchecking Auto check for update or Load at startup makes the Start and Stop buttons light up again but not always.

Reset size in the right click menu also does not seem to work.

Another thing that doesn't work well is exiting the app. Right clicking on the task bar/quick access icon and clicking Exit does not seem to work at all.

I can provide anything you need - let me know, thanks.

Thanks!

**Comment by just4747** [comment]
Hi - have tried everything, reinstalled, messed with all settings and I still cannot get this program to do/change anything for my cross-monitor mouse alignment at all. Was working perfectly on old version (aside from auto-start). Also, start/stop buttons rarely able to be pressed in main UI at all, cannot even tell if it is actually started or not/state it is in. Please let me know what I can try or if I can provide anything for you to look into. Thanks.

**Comment by just4747** [comment]
This app does not work at all for me (on newest 5.0.5.0). I have tried everything. Old version worked fine for me. Can you please take a look? I can send log, etc... Thanks.

**Comment by mgth** [comment]
Hi, I added a global catch in 5.0.5.0, you did not get any message when launching LittleBigMouse.exe ? If so you can copy details from there.

**Comment by just4747** [comment]
> Hi, I added a global catch in 5.0.5.0, you did not get any message when launching LittleBigMouse.exe ? If so you can copy details from there.

Hi - I'm pretty sure I got the message and OKd it. Is there anything else I can check? It's as if the program settings are just being ignored. Nothing changes when activated. Also very very sporadic being able to even press Start and Stop... definitely seems to be semi-broken (at least for me).

**Comment by just4747** [comment]
[LittleBigMouse.Export.gz](https://github.com/mgth/LittleBigMouse/files/13732397/LittleBigMouse.Export.gz)

Attached config here but not sure if that helps.

**Comment by permster** [comment]
Similar issues here with 5.0.5.0 and a previous version of 5.x.  The main two things I've noticed are that under Size, "reset size" and "place from windows config" options do nothing.

The whole resize thing in general could use some help.  It's far too easy to accidentally put the wrong dimensions in due to the way the dimension input fields work.  After it's saved you can't adjust the sizes in the UI if the monitor stretches outside the LBM window or when the monitor is too tiny.  Once you mess it up your left with going into the registry to fix it.

**Comment by just4747** [comment]
> Similar issues here with 5.0.5.0 and a previous version of 5.x. The main two things I've noticed are that under Size, "reset size" and "place from windows config" options do nothing.
> 
> The whole resize thing in general could use some help. It's far too easy to accidentally put the wrong dimensions in due to the way the dimension input fields work. After it's saved you can't adjust the sizes in the UI if the monitor stretches outside the LBM window or when the monitor is too tiny. Once you mess it up your left with going into the registry to fix it.

Yep essentially nothing at all work for me. No settings, virtually no buttons, no configuration changes with the actual monitors, nothing seems to be functioning.

**Comment by just4747** [comment]
@mgth also when I try to activate the program via command line, it gives me this error:

"This app can't run on your PC. To find a version for your PC, check with the software publisher" 

See attached image. Is this an indication that maybe the app is never running on my machine to begin with, hence why I am not seeing any changes happening?

![Screenshot 2023-12-27 135359](https://github.com/mgth/LittleBigMouse/assets/558589/9fc032c4-31fc-4877-bac3-e68741644f69)

**Comment by just4747** [comment]
Hi @mgth - thanks so much for all your work so far. Really eager to get this working again. Are you able to see my last post with the image - I can provide anything else that may help on this - still unable to get it working at all. Thank you!

**Comment by just4747** [comment]
still nothing. app doesnt work at all. downgrading to 2019 version now, hope it works. new versions are useless sadly (at least for me).

**Comment by mgth** [comment]
Housekeeping pass: this report predates the full v5 rewrite of the mouse engine, and 5.3.0/5.4.x fixed a large batch of long-standing issues (stuck edges, silent startup failures, duplicated displays, sleep/resume, DPI handling…). This exact setup is unlikely to behave the same today, so I'm closing old reports in bulk.

If you still hit this on [5.4.1](https://github.com/mgth/LittleBigMouse/releases/tag/v5.4.1), a comment reopens it — fresh reports are always welcome.
